docs(www): fix duplicate h1's from markdown and frontmatter (#15094)
gatsbyjs/gatsby: docs/docs/mdx/writing-pages.md

> **Note**: the default export concept used in this code block is explained in more detail
> in the docs below on [defining layouts](#defining-a-layout)

## Combining frontmatter and imports

> If you would like to include frontmatter and import components, frontmatter needs to appear at the top of the file, imports can then follow after it:

Reviewer: ```suggestion
If you would like to include frontmatter metadata _and_ import components, the frontmatter needs to appear at the top of the file and then imports can follow:
```

```mdx
---

> title: "Building with Gatsby"

Reviewer: I think the quotes can be removed from this one, too.

Author: ```suggestion
title: Building with Gatsby
```
